opendap_tests.py

```diff
@@ -171,6 +179,14 @@ def save_response(url_address, ext, r):
         base_name = save + '/' + base_name
     with open(base_name + ext, 'w') as file:
         file.write(r.text)
+    with open(base_name + ext + ".h", 'w') as header:
```

Comment:
I think we should keep this in the code. If we are saving the response, lets also keep the headers. Good call.